plugins/cybereason/plugin.spec.yaml

```diff
@@ -502,37 +502,37 @@ actions:
         required: true
   isolate_machine:
     title: Isolate Machine
-    description: Isolate a machine associated with the root cause of a Malop, or to remediate a process not involved in a Malop
+    description: Isolate or un-isolate a machine
     input:
+      sensor:
+        title: Sensor
+        description: Sensor ID, hostname or IP address of the sensor to perform the action on
+        type: string
+        required: true
+        example: 198.51.100.100
+      quarantine_state:
+        title: Quarantine State
+        description: True to isolate the sensor, false to un-isolate it
+        type: boolean
+        required: true
+        default: true
+        example: true
       malop_id:
         title: Malop ID
-        description: Malop ID to isolate a machine or empty to remediate process not involved in a Malop
+        description: Malop ID to associate with the quarantine action
         type: string
         required: false
         example: "22.2787422324806222966"
-      pylum_ids:
```

Comment:
Did we mean to remove these? If so, the plugin needs a major version bump. If a customer used the current version of the plugin in a workflow and relied on the keys in the workflow logic and updated to this one, it would fail and thus not be backward-compatible.

Reply:
Thanks for picking up on this. Correct, I updated the functionality of the isolate machine, and moved the rest of the functionality to `Remediate Items` action, therefore it's a breaking change. I've bumped the major version number for this change.